db/Dockerfile

```diff
@@ -5,4 +5,7 @@ COPY *.conf /data/
 COPY sql/* /docker-entrypoint-initdb.d/
 RUN chmod a+r /docker-entrypoint-initdb.d/*
 
+COPY seed/* /seed/
+RUN chmod a+r /seed/*
```

Comment:
Are these chmod steps necessary? It looks like the postgres container container gets run with the `root` user, so there shouldn't be any permissions issues :thinking:

Reply:
Unsure I can try it out I guess. Doesn't really matter either way, though, I don't think.

Reply:
You are correct, have removed.

Reply:
Pog